tests/safety/test_subaru_preglobal.py

```diff
@@ -61,5 +61,12 @@ def _pcm_status_msg(self, enable):
     return self.packer.make_can_msg_panda("CruiseControl", 0, values)
 
 
+class TestSubaruPreglobalReversedDriverTorqueSafety(TestSubaruPreglobalSafety):
+  FLAGS = Panda.FLAG_SUBARU_PREGLOBAL_REVERSED_DRIVER_TORQUE
+
+  def _torque_driver_msg(self, torque):
+    return super()._torque_cmd_msg(-torque)
```

Comment:
@jnewb1 the tests don't fail if you remove the sign from here

Reply:
fixed by using correct DBC